### DISCUSSION TITLE
没有“局域网共享”选项

### DISCUSSION THREAD

**Discussion by Perter1999** [v2rayA/v2rayA#1397] [Q&A] [answered]
![Snipaste_2024-05-05_23-22-18](https://github.com/v2rayA/v2rayA/assets/55939736/66721109-ff2d-45a5-961b-bcf99dd969ed)
我采用的是docker安装，最新拉取的镜像。我想给局域网其它设备使用，但是没有“局域网共享”选项。我看有人提过历史问题，但是已经找不到该问题了。https://github.com/v2rayA/v2rayA/issues/741

**Comment by zuilintan** [comment, marked as answer]
开启端口分享
就是我们所需要的

**Reply by cnazev** [reply]
多谢 那个开启ip转发又是做什么用？

**Reply by pcmcon** [reply]
> 多谢 那个开启ip转发又是做什么用？

网关代理

**Reply by tnessn** [reply]
开启之后怎么连的 为啥我用sock5连不上 端口是哪个

**Comment by github-office** [comment]
各位老铁，sock5能设置密码嘛？

**Comment by Perter1999** [comment]
谢谢，现在暂时不用了
